### ISSUE TITLE
Manual intervention when using TLS enabled DB on MySQL for PCF 2.4.x

### ISSUE THREAD

**Issue by csterwa** [pivotal-cf/docs-spring-cloud-dataflow#58]
From Tao Deng:

@csterling @judy

I tested SCDF for PCF v1.3.2 internal release in support team lab 16. The deploy-broker errand fails due to error:

```
         cf service p-dataflow-db shows 'status:    create in progress'. Waiting...
         cf service p-dataflow-db shows 'status:    create succeeded'
         cf bind-service p-dataflow-1.3.2-build.1 p-dataflow-db
         Binding service p-dataflow-db to app p-dataflow-1.3.2-build.1 in org system / space p-dataflow as system_services...
         Service broker error: This service instance can only be used with TLS enabled. In order to enable TLS for this service instance run "cf update-service SERVICE_INSTANCE -c '{"enable_tls": [HOSTNAME]}'"
         FAILED
```

Because MySQL TLS instance must be manually enabled with `cf update-service p-dataflow-db -c '{"enable_tls": ["xxxx.xxxxx.bosh"]}'`
https://docs.pivotal.io/p-mysql/2-4/using-tls.html

Once update instance manually, both broker deployment and smoke-test succeeded.

SCS docs should include this as informational for installation and pre-requisites or some other section to notify before install.

**Comment by fifthposition** [comment]
This is up at:

* http://docs-pcf-staging.cfapps.io/scdf/1-3/index.html#reqs
* http://docs-pcf-staging.cfapps.io/scdf/1-3/installing.html#using-tls-with-mysql-v2-4-x

**Comment by csterwa** [comment]
@fifthposition should there be a section in the troubleshooting section, as well?

**Comment by csterwa** [comment]
@fifthposition as discussed offline, I think we should add this as known issue that can impact SCDF for PCF on the release notes that is resolved with 1.3.2 when release notes are created for that.

**Comment by fifthposition** [comment]
Added 'Known Issues' that points to the Installation page's section.

http://docs-pcf-staging.cfapps.io/scdf/1-3/release-notes.html#known-issues
